WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -19,14 +19,41 @@ <h2>Help Pages</h2>
 
     <section class="help-topic panel panel-default" data-topics="link reduced data">
         <div class="panel-heading">
-            <h4>How do I use the link to reduced data?</h4>
+            <h4>How do I access my reduced data?</h4>
         </div>
         <div class="panel-body">
             <p>
-                The link to reduced data points to a network drive that is accessible from within the RAL site (including by VPN).
-                On Windows, copy the link into the navigation bar at the top of a File Explorer window and hit enter.
-                Alternatively, the network drive (<code>\\data.analysis.stfc.ac.uk\rb\</code>) can be mapped to a location on your disk.
+                In order to copy reduced data from CEPH to your local machine you will need to download an SFTP client.
+                We recommend <a href="https://winscp.net/eng/download.php">WinSCP</a> if you are using a windows operating system.
+                The step below are based on WinSCP as the <code>SFTP Client</code> and assume you have downloaded and installed it.
             </p>
+            <h2>IDAaaS</h2>
+            <ol>
+                <li>Visit the IDAaaS page <a href="https://isis.analysis.stfc.ac.uk">https://isis.analysis.stfc.ac.uk</a></li>
+                <li>Create a new IDAaaS instance for your instrument / technique.</li>
+                <li>Follow the instructions on the <a href="https://isis.analysis.stfc.ac.uk/#/help">IDAaaS help page</a>
+                    under the <em>Download/Upload Files</em> section that will walk you through the SFTP process </li>
+            </ol>
+            <h2>ISIS Compute</h2>
+            If you do not currently have your instrument supported by IDAaaS then you should use the ISIS Compute service.
```

Comment:
Is this option accessible from outside the site firewall?

Reply:
I don't believe so. I have now added this to the top section stating that you must be onsite/VPN to access